artic/minion.py

```diff
@@ -26,13 +28,24 @@ def get_nanopolish_header(ref):
         raise SystemExit(1)
     return  "%s:%d-%d" % (recs[0].id, 1, len(recs[0])+1)
 
+def check_scheme_hashes(filepath, manifest_hash):
+    with open(filepath, "rb") as fh:
+        data = fh.read()
```

Comment:
I imagine the scheme is reasonably small, but note that this method will load all the contents of filepath to memory -- so consider chunking.

Reply:
You're right but since we control the manifest.json and as such the files which get downloaded no files larger than 1M should ever be downloaded here